packages/shared/src/store/activityActions.ts

```diff
@@ -27,7 +27,12 @@ export async function muteChat(chat: db.Chat) {
     const volume = ub.getVolumeMap(muteLevel, true);
     await api.adjustVolumeSetting(source, volume);
   } catch (e) {
-    logger.log(`failed to mute chat ${chat.id}`, e);
+    logger.trackError('ActivityAction: Failed to mute chat', {
+      chatId: chat.id,
```

Comment:
For now we still need to manually wrap sensitive info like this with `logic.getModelAnalytics`